jdaviz/configs/specviz2d/tests/test_parsers.py

```diff
@@ -134,6 +137,25 @@ def test_2d_1d_parser(specviz2d_helper, mos_spectrum2d, spectrum1d):
     specviz2d_helper.load_data(spectrum_2d=mos_spectrum2d, spectrum_1d=spectrum1d)
     assert specviz2d_helper.app.data_collection.labels == ['Spectrum 2D', 'Spectrum 1D']
 
+    spec2d_viewer = specviz2d_helper.app.get_viewer("spectrum-2d-viewer")
+    assert spec2d_viewer.figure.axes[0].label == "x: pixels"  # -0.5, 14.5
+    spec2d_viewer.apply_roi(XRangeROI(10, 12))

```

Comment:
Could have used the API from #3104 but since it is not merged yet, I didn't.

Reply:
https://github.com/spacetelescope/jdaviz/pull/3104 is now merged but the API is not yet public. Should I keep this as-is or should I use the `helper.plugins['Subset Tools']._obj.import_region(...)` syntax here?